Spec/README.md

```diff
@@ -244,29 +244,30 @@ The filename must not contain anything else.
 
 ### Supported Styles
 
-Google’s static fonts API supports up to 18 styles in one family: up to 9 weights (Thin–Black), + their matching Italics. The table below lists each style its font specific settings. *TODO MF convert integers to bits*
+Google’s static fonts API supports up to 18 styles in one family: up to 9 weights (Thin–Black), + their matching Italics. Bear in mind the API will automatically create all the 100x weights along the range of the axis when producing the static fonts.
```

Comment:
> Bear in mind the API will automatically create all the 100x weights along the range of the axis when producing the static fonts.

This statement should be included in the VF section. We're not talking about instantiated statics here.

Reply:
Fixed in 0634e02